compiler/rustc_llvm/llvm-wrapper/PassWrapper.cpp

```diff
@@ -350,6 +356,7 @@ void LLVMRustAddLastExtensionPasses(
   SUBTARGET_ARM                                                                \
   SUBTARGET_AARCH64                                                            \
   SUBTARGET_AVR                                                                \
+  SUBTARGET_LOONGARCH
```

Comment:
```suggestion
  SUBTARGET_LOONGARCH \
```

I guess this breaks CI.